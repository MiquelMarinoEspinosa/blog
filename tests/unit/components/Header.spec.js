import Vue from "vue";
import { shallowMount } from "@vue/test-utils";
import Header from "@/components/Header";
import VueRouter from "vue-router";

Vue.use(VueRouter);

describe("Header nav links", () => {
  let header;
  beforeEach(() => {
    header = shallowMount(Header);
  });
  afterEach(() => {
    header = null;
  });

  it("renders the inital html", () => {
    expect(header.html()).toMatchSnapshot();
  });
});
