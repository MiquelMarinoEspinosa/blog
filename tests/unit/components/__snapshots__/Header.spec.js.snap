// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header nav links renders the inital html 1`] = `
<nav>
  <ul>
    <li>
      <router-link-stub to="/" tag="a" exact="true" ariacurrentvalue="page" event="click">Blog</router-link-stub>
    </li>
    <li>
      <router-link-stub to="/add" tag="a" exact="true" ariacurrentvalue="page" event="click">Add a New Blog</router-link-stub>
    </li>
  </ul>
</nav>
`;
